sdk/nodejs/runtime/closure/codePaths.ts

```diff
@@ -270,24 +258,8 @@ async function allFoldersForPackages(
     // Read package tree from the workspace root to ensure we can find all
     // packages in the workspace.  We then call addPackageAndDependenciesToSet
     // to recursively add all the dependencies of the referenced packages.
-    const root = await pReadPackageTree(workspaceRoot);
-
-    // read-package-tree defers to read-package-json to parse the project.json file. If that
-    // fails, root.error is set to the underlying error.  Unfortunately, read-package-json is
-    // very finicky and can fail for reasons that are not relevant to us.  For example, it
-    // can fail if a "version" string is not a legal semver.  We still want to proceed here
-    // as this is not an actual problem for determining the set of dependencies.
-    if (root.error) {
```

Comment:
Arborist does not have the same issue, it is much more lenient with parsing package.json. We have a test case for this and it stil passes with arborist.